### ISSUE TITLE
Changing any of the stick curves doesn't apply at all! 

### ISSUE THREAD

**Issue by GittyUser54321** [steffalon/dualsense-edge-profile-web-application#5]
As per title! it would be great if it was working if you could include the graphic and how it changes the curves like in ps5: 

![DSE sens curves](https://github.com/steffalon/dualsense-edge-profile-web-application/assets/101957276/27015018-cc38-4e3d-8ce8-6af8b70849d6)

![DSE settings](https://github.com/steffalon/dualsense-edge-profile-web-application/assets/101957276/d4e088f1-ece0-4d6d-b9a6-48e983e2d763)

Oh and to take it a step further, it would be great to be able to add a custom curve not already in the presets. Similar to how DS4 does it with their BezierCurveEditor: https://github.com/gre/bezier-easing-editor/tree/master/example

**Comment by steffalon** [comment]
Hi @GittyUser54321, Thank you for your suggestion. changing the curves are fixed and will apply correctly (as you mentioned before in ticket #6).

The BezierCurveEditor is very outdated and built for React instead of Vue. Instead, I will make something from scratch. I expect to release this feature today.

**Comment by steffalon** [comment]
Let me know if this issue is fully resolved :)

**Comment by GittyUser54321** [comment]
> Let me know if this issue is fully resolved :)

Ok thank you. So this means that you can't do custom curves? Also, what does the red lines mean on the graphic? Hoping its not the deadzones cos you can't adjust those on there (would prefer 0 anyways).

Also, not sure if you are aware and you may need to test it again but the button remapping wasn't working after the curve updates.

**Comment by steffalon** [comment]
Red line is to test your input. All to left === full stick input. 

"Also, not sure if you are aware and you may need to test it again but the button remapping wasn't working after the curve updates." Mine still works? Even after changing curve values and curve type. Could you reproduce your steps?

**Comment by steffalon** [comment]
"So this means that you can't do custom curves?" it is possible however the PS5 does not allow to make custom values ouside their presets. I'm not sure how much I can customize it regarding the curve positions.

**Comment by GittyUser54321** [comment]
> Red line is to test your input. All to left === full stick input.
> 
> "Also, not sure if you are aware and you may need to test it again but the button remapping wasn't working after the curve updates." Mine still works? Even after changing curve values and curve type.

Yeah any button remapping doesn't work for me at the moment. Maybe its because I used DSX on steam to remap the buttons to my profiles there. But thats weird cos it shouldn't interfer with this webapp after I have closed that app off. It only started happening when the curves got fixed cos before it didn't matter if I used DSX as I could go on your webapp and still change the remappings.

**Comment by GittyUser54321** [comment]
> "So this means that you can't do custom curves?" it is possible however the PS5 does not allow to make custom values ouside their presets. I'm not sure how much I can customize it regarding the curve positions.

Oh ok hmmm. Thats a bummer. It would have been epic to be able to really do your own curves outside of the preset ones and save them to the controller.

**Comment by steffalon** [comment]
I could experiment setting custom values. But I do need to redesign some functions. So it might not be a quick fix.

**Comment by steffalon** [comment]
Are you able to change something on the controller using this app? Also, make sure you are using usb instead of bluetooth.

**Comment by GittyUser54321** [comment]
> Are you able to change something on the controller using this app? Also, make sure you are using usb instead of bluetooth.

Ok so I cleared the profile I was using and started again on this webapp and all is working as usual. Must have been a bug when using DSX along with this app.

**Comment by GittyUser54321** [comment]
> I could experiment setting custom values. But I do need to redesign some functions. So it might not be a quick fix.

Yeah honestly this would be amazing. Alot of games have weird curves and being able to do custom curves to counteract them would be super useful. And even just custom curves in general. Would make this app better than even the Ps5 one.

**Comment by steffalon** [comment]
I'll try fixing it in the future :) But the DSX conflicting this app is very odd. Not sure why it would not save input mappings.

**Comment by GittyUser54321** [comment]
> I'll try fixing it in the future :) But the DSX conflicting this app is very odd. Not sure why it would not save input mappings.

Yeah super strange bug. Thanks a lot for everything.
